packages/asset-uploader/src/db/create.ts

```diff
@@ -46,91 +37,21 @@ export const createAssetWithLimit = async (
     throw new Error("You don't have access to create this project assets");
   }
 
-  try {
-    /**
-     * sometimes for example on request timeout we don't know what happened to the "UPLOADING" asset,
-     * so we don't take into account assets with the "UPLOADING" status that were created more
-     * than UPLOADING_STALE_TIMEOUT milliseconds ago
-     **/
-    const UPLOADING_STALE_TIMEOUT = 1000 * 60 * 30; // 30 minutes
-
-    const count = await prisma.asset.count({
-      where: {
-        OR: [
-          { projectId, status: "UPLOADED" },
-          {
-            projectId,
-            status: "UPLOADING",
-            createdAt: { gt: new Date(Date.now() - UPLOADING_STALE_TIMEOUT) },
-          },
-        ],
-      },
-    });
-
-    if (count >= env.MAX_ASSETS_PER_PROJECT) {
```

Comment:
Do we still check limit at the server side somewhere? Need to make sure we check as client-side check is easy to bypass.

Reply:
We can do truncation in patches, wdyt?

Reply:
Sounds good